extensions/commands/art/cmd_build_info.py

```diff
@@ -209,22 +220,9 @@ def _get_remote_artifacts(artifact):
 
             return artifact_info
 
-        artifacts, missing = _get_local_artifacts()
-
-        if 'conan_sources.tgz' in missing:
```

Comment:
maybe show a warning message and check artifactory if credentials are provided but do not bother user with artifactory credentials for nothing